db/test-db/schema.sql

```diff
@@ -0,0 +1,53 @@
+CREATE DATABASE cloud_ecommerce WITH TEMPLATE = template0 ENCODING = 'UTF8' LC_COLLATE = 'en_US.utf8' LC_CTYPE = 'en_US.utf8';
+\connect cloud_ecommerce
+create table products (
+  pid serial primary key,
+  pcode varchar(50),
+  price float(2) check (price is null or price >= 0),
+  sku varchar(50) not null,
+  amount_in_stock int check(amount_in_stock >= 0),
+  pname varchar(50) not null,
+  description text not null,
+  lang varchar(7) not null
+);
+
+create table users (
+  uid varchar(20) primary key,
+  fname varchar(50) not null,
+  lname varchar(50) not null,
+  address varchar(50) not null,
+  phone numeric(10) not null, -- 10 digits
```

Comment:
add a constraint that checks for 10 digits when a record is inserted:
check (phone between 0000000000 and 9999999999)